src/main/scripts/summa_export/get_records.sh

```diff
@@ -58,15 +63,28 @@ get_record() {
     echo " - Fetching $OUT_FOLDER/$OUT_NAME"
     if [[ "true" == "$EXPAND" ]]; then
         # expand is ignored
-        curl -s -G "$SUMMA_STORAGE?method=getCustomRecord&expand=false&legacyMerge=false&escapeContent=false&" --data-urlencode "id=${RECORD_ID}" | sed -e 's/<soapenv.*getCustomRecordReturn[^>]*>//' -e 's/<\/ns1:getCustomRecordReturn.*//' -e 's/&lt;/</g' -e 's/&gt;/>/g' -e 's/&quot;/"/g' -e 's/&amp;/&/g' | xmllint --format - > "$OUT_FOLDER/$OUT_NAME"
+
+        # Fetch the raw response (entity escaped record XML wrapped in a SOAP envelope)
+        local RESPONSE=$(curl -s -G "$SUMMA_STORAGE?method=getCustomRecord&expand=false&legacyMerge=false&escapeContent=false&" --data-urlencode "id=${RECORD_ID}")
+
+        # Check that there is content inside of the SOAP envelope
+        if [[ "." != ".$( tr '\n' ' ' <<< "$RESPONSE" | grep "<ns1:getCustomRecordReturn[^>]*>.\{10,\}<\/ns1:getCustomRecordReturn")" ]]; then
+            echo "$RESPONSE" | sed -e 's/<soapenv.*<ns1:getCustomRecordReturn[^>]*>//' -e 's/<\/ns1:getCustomRecordReturn.*//' -e 's/&lt;/</g' -e 's/&gt;/>/g' -e 's/&quot;/"/g' -e 's/&amp;/&/g' | xmllint --format - > "$OUT_FOLDER/$OUT_NAME"
+        else
+            >&2 echo "Error: The record with the ID '$RECORD_ID' could not be fetched"
+            echo "$RECORD_ID" >> "$OUT_FOLDER/failed.dat"
```

Comment:
This file will list a number of unexported IDs, but it will be impossible to see the run in which they failed. This will make it hard to know how far back to do cleanup, since the file is forever appended to.
Also, what to do with this file after cleanup? At least instructions for how to handle these circumstances would be helpful - I guess the answer is "delete the file after cleanup. Make sure the export is not running at the same time"

Reply:
Since the scripts around `get_records.sh` looks for zero-sized output files and files them as failed, maybe it is a mistake to make a dedicated file with those IDs? Should I just remove this part of the pull-request and let the script create the zero-size file instead?

Reply:
The script around it doesn't recognise exactly WHY transformation fails, but 0-size files is one reason.
In practice it fails them because it cannot find a valid channel id.
But yes, I think it would probably be best to handle failed files in the same way. Perhaps we could add an explicit check for 0-size-files in the surrounding script instead?
I do think we should keep the code recognising unfound data though, to get a reasonable log message